src/main/java/com/cloudogu/wiki/WikiDispatcherServlet.java

```diff
@@ -78,6 +81,46 @@ protected void service(HttpServletRequest req, HttpServletResponse resp) throws
         }
     }
 
+    public List<Group> sortingWikisByGroups(Iterable<Wiki> wikis) {
+
+            /* getGroups gives a list with the groups including their repos.
+             * Therefore, the Wiki list (current order: 'repo 1 (group a), repo 2 (group a), ...')
+             * needs to be splitted into an order as 'group a (repo 1, ..), group b (repo x, ..), ..',
+             * that desired presentation in overviewRepos_en/de.html is possible.
+             */
+
+        String currentGroup = "";
+        List<Wiki> currentRepos = new ArrayList<Wiki>();
+        List<Group> groups = new ArrayList<Group>();
+
+        if(Iterables.size(wikis) != 0) {
+
+            for (Wiki wiki : wikis) { // wikis are already sorted by group name and alphabetically in ScmManager.getPotentialWikis
+
+                if (currentGroup != null && currentGroup.equals("")) { // first iteration -> currentGroup needs to get the current groupname
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the "" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS1132)